commands/utility/feedback.py

```
@@ -78,6 +78,7 @@ async def on_submit(self, interaction: discord.Interaction) -> None:
         )
 
         embed = tanjunEmbed(
+            colour=EmbedColor.SUCCESS,
```

Flake8 (7.3.0)

[error] undefined name 'EmbedColor'

(F821)